### PR TITLE
Fix #4 -Add line that notify user about short cuts

### LINKED ISSUE #4: Adding a line to inform user about the Ctrl+X short cut
Would like to add one line where it will say about the shortcut that saves the updated note

### PR DESCRIPTION
**Pull request by lozinska**
Add line that notify user about short cut

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,8 +34,10 @@ <h2>Welcome to mini note!</h2>
         else console.log("Saved");
       });
     });
-
   </script>
+  <p style="color: red; text-align:centre;font-weight: bolder;">
+    To save use CTRL+S
+  </p>
```
Comment (manekenpix) [inline index.html]: I think it's better if you add this `<p>` tag before my `<script>` tags, right after my `<div id=note>` tag.

Comment (lozinska) [reply index.html]: OK sounds good, i'll fix the issue

**Comment by manekenpix** [comment]
@lozinska I merged a pull request from someone else. I think it'd be a good idea to incorporate that new code to your fork.

**Comment by lozinska** [comment]
So you want to keep as it is, just let someone else to fix that issue?

**Comment by lozinska** [comment]
I updated the html file, if you can please let me know if anything else need to be done. Thank you

**Comment by manekenpix** [comment]
This looks great, I was just suggesting that, since someone else added new code to the website, maybe it would be a good idea for you to also have that code so you can work with the latest updates of the main website.

**Pull request closed without merging**